doc/nrf/releases/release-notes-2.0.0.rst

```diff
@@ -0,0 +1,1091 @@
+  * :ref:`Zigbee shell library <lib_zigbee_shell>`.
+    For details, see `Libraries for Zigbee`_.
+
+* Fixed:
+
+  * An issue where the Light Bulb sample was flickering when set to 50 kHz.
+  * An issue with an assertion fail in :file:`/zephyr/include/spinlock.h:129`.
+  * An issue where a wrong value was reported for attributes MinMeasuredvalue and MaxMeasuredValue in the Weather Station.
+  * An issue with ZBOSS fatal error after factory reset and before bdb start.
+  * An issue where the Coordinator did not form a new network after factory reset.
+  * An issue with identifying when not in a network.
+  * An issue with ZBOSS binding table corruption.
+  * An issue where the Zigbee shell did not inform if the network address request was not sent.
+
+See `Zigbee samples`_ for the list of changes for the Zigbee samples.
+
+HomeKit
+-------
+
+* Added
+
+  * Production support for Thread 1.2.
+  * Support for 3-button actions (long press, short press, double press).
+  * LED indicating BLE connectivity status.
+  * Development support for OTA-DFU using the iOS Home App (over UARP - BLE and Thread).
+  * LED output can be fully disabled via kConfig.
+
+* Updated
+
+  * Flash usage optimization for debug versions of samples and applications.
+  * DFU mode can now be enabled by a button press.
+  * CLI is no longer required for DFU configuration.
+  * HAP_TESTING is now configurable via kConfig.
+
+* Fixed
+
+  * An issue where RTT logs did not work with the Light Bulb multiprotocol sample with DFU on nRF52840.
+  * An issue where Nordic DFU was not compliant with HAP certification requirements.
+  * An issue where a change in KVS key naming scheme caused an error for updated devices.
+  * An issue where activating DFU caused increased power consumption.
+
+Applications
+============
+
+This section provides detailed lists of changes by :ref:`application <applications>`.
+
+nRF9160: Asset Tracker v2
+-------------------------
+
+* Added:
+
+  * Support for :ref:`Bosch Software Environmental Cluster (BSEC) library <bosch_software_environmental_cluster_library>` driver.
+  * Support for Indoor Air Quality (IAQ) readings retrieved from the BME680 sensor on Thingy:91.
+    For more information, see the :ref:`asset_tracker_v2_sensor_module`.
+  * Support for QEMU x86 emulation.
+  * Support for the :ref:`lib_nrf_cloud_pgps` flash memory partition under certain conditions.
+  * Support for :ref:`QoS` library to handle multiple in-flight messages for MQTT based cloud backends such as AWS IoT, Azure IoT Hub, and nRF Cloud.
+  * Support for Lightweight Machine to Machine (LwM2M).
+  * Support for filtering updates to cloud based on LTE connection evaluation (`AT%CONEVAL`_).
+
+* Updated:
+
+  * For nRF Cloud builds, the configuration section in the shadow is now initialized during the cloud connection process.
+  * The :ref:`lib_nrf_cloud` library can now handle modem FOTA updates if :kconfig:option:`CONFIG_NRF_CLOUD_FOTA` is enabled.
```

Comment:
There was an important change as part of the P-GPS flash memory partition PR. This should be included in the release notes:

```suggestion
  * The :ref:`ug_bootloader` has been enabled by default. To diable, set the :kconfig:option:`CONFIG_SECURE_BOOT` Kconfig option to ``n``.
  * The :ref:`lib_nrf_cloud` library can now handle modem FOTA updates if :kconfig:option:`CONFIG_NRF_CLOUD_FOTA` is enabled.
```

Reply:
small typo. Missing s in disable

Reply:
done